make/kube

```diff
@@ -6,15 +6,18 @@ GIT_ROOT=${GIT_ROOT:-$(git rev-parse --show-toplevel)}
 cd "${GIT_ROOT}"
 source .envrc
 
-CREATE_HELM_CHART=false
-BUILD_TARGET=kube
+DOMAIN=${DOMAIN:-}
+if test -n "$DOMAIN"; then
+    # If DOMAIN is set, place DOMAIN and UAA_HOST at the end of network.env temporarily
+    echo "DOMAIN=${DOMAIN}" >> bin/settings/network.env
+    echo "UAA_HOST=uaa.${DOMAIN}" >> bin/settings/network.env
+    trap "sed -i '$d' bin/settings.network.env && sed -i '$d' bin/settings/network.env" EXIT
```

Comment:
/home/vagrant/scf/src/make/kube: line 14: d: unbound variable
The outer `"`-quoting causes interpolation of `$d`, `d` does not exist.
Has to be `\`-escaped.
Gr. Still not working. No error anymore, but no deletion of lines either.
Because: `sed: can't read bin/settings.network.env: No such file or directory`
The sed seems to run in a different directory context.